tests/foreman/ui/test_hostgroup.py

```diff
@@ -190,6 +190,47 @@ def test_positive_create_with_activation_keys(self):
 
     @run_only_on('sat')
     @tier1
+    def test_positive_redirection_for_multiple_hosts(self):
+        """Create new hostgroup with whitespaces in its name for specific
+        organization and location. After that create some hosts using already
+        created hostgroup
+
+        :id: f301ec2f-1e9b-4e34-8c87-4d0b3381d9e1
+
+        :expectedresults: Hostgroup hosts count can be successfully observed
+            in a table and redirection to these hosts works properly
```

Comment:
Got it now, thanks. Was initially thinking it's some kind of 'Hosts' tab under specific hostgroup, for that case redirection to host page would be a good thing to test (as we often have bugs in such links).